nautobot/docs/development/core/model-checklist.md

```diff
@@ -34,6 +34,22 @@ article ul li {
     - Update ConfigContextQuerySet, ConfigContextFilterSet, ConfigContext forms, edit and detail views and HTML templates
 - _optional_ Expose any new relevant Python APIs in `nautobot.apps` namespace for App consumption
 
+#### Extras Features
```

Comment:
Probably the mention of `extras_features` at line 23 above should link to this heading.